public/locales/en/translation.json

```diff
@@ -19,6 +19,11 @@
     }
   },
   "home": {
+    "genderViolenceMain": {
+      "header": "You are not alone {{break}} Safe Place is {{span}}",
```

Comment:
still the word span is making translations know about html. break even more

you want to split this into two parts. one for You are not alone. one for the other. put the br in the html.

then give span a meaningful name
the way this works is that translation files are usually handed to non code knowing translators. so the placeholder needs to be meaningful to them.

Reply:
so I need to have "headerPartOne" and headerPartTwo and break is in html. while the span will stay in the same way but with meaningful name. the span is just used to styling the word here at the end of the sentence in english